python/src/Collection_doc.i.in

```diff
@@ -67,7 +67,13 @@ Examples
 [Normal(mu = 0, sigma = 1),Uniform(a = -0.5, b = 1)]
 >>> vector.add(ot.Gamma(2.75, 1.0))
 >>> print(vector)
-[Normal(mu = 0, sigma = 1),Uniform(a = -0.5, b = 1),Gamma(k = 2.75, lambda = 1, gamma = 0)]"
+[Normal(mu = 0, sigma = 1),Uniform(a = -0.5, b = 1),Gamma(k = 2.75, lambda = 1, gamma = 0)]
+
+- Collection of **design of experiments**:
+
+>>> collection = ot.WeightedExperimentCollection(2)
+>>> collection[0] = ot.GaussProductExperiment(ot.Uniform(0.0, 1.0), [3])
+>>> collection[1] = ot.GaussProductExperiment(ot.Uniform(0.0, 1.0), [5])"
```

Comment:
same remark here, we should not have to expose WeightedExperimentCollection in python

Reply:
Ok, I remove the example. But I do not understand why: after all, the same principle could be applied to the `DistributionCollection` and the other Collection class, which are documented in the same page, isn'it? Shall I remove the whole page?

Reply:
yes, DistributionCollection should go too

Reply:
Then the whole page must be removed, because none of these examples are appropriate in the Python setting. I remove the python/src/Collection_doc.i.in file then.

Reply:
Fixed at e241ee5.

Reply:
```
$ grep -nr Collection ../python/doc/user_manual/
../python/doc/user_manual/base_objects.rst:92:    BoolCollection
../python/doc/user_manual/base_objects.rst:95:    IndicesCollection
../python/doc/user_manual/base_objects.rst:98:    ComplexCollection
../python/doc/user_manual/base_objects.rst:99:    ScalarCollection
../python/doc/user_manual/probabilistic_modelling.rst:17:    DistributionCollection
```

Reply:
Ok for these entries. But there are many existing unit tests using DistributionCollection:
```
$ grep -rw "DistributionCollection" ../python/ | wc -l
84
```
Many of these are in unit tests. I try to remove as many as I can.

Reply:
The `MarginalTransformationEvaluation::getInputDistributionCollection` returns a `DistributionCollection`. Should I replace this by "sequence of `Distribution`"? The question is: OT will return a `DistributionCollection`, and not a list. Then the user will be given an undocumented object. Is is this correct?

Reply:
Up to now, the rule is: for input arguments, mention the most pythonish container accepted by the method (e.g. a list of scalars instead of a Point) but the most accurate type for the returned values (so a DistributionCollection if needed). And IMO these collections should be documented.

Reply:
Oups. Then we should discuss that during the next TC.

Reply:
Taking care of the Collection objects everywhere can be done in another PR, it should not stop you from doing this.